pkg/api/v20231122/openshiftcluster.go

```diff
@@ -50,7 +50,7 @@ type OpenShiftClusterProperties struct {
 	ClusterProfile ClusterProfile `json:"clusterProfile,omitempty"`
 
 	// The console profile.
-	ConsoleProfile ConsoleProfile `json:"consoleProfile,omitempty"`
+	ConsoleProfile ConsoleProfile `json:"consoleProfile,omitempty" swagger:"readOnly"`
```

Comment:
I think instead of making the whole `Profile` readonly, we just want the URL (which, yea, is the only thing in there... but... (see L122)